site/content/docs/5.3/examples/_index.md

```diff
@@ -6,88 +6,3 @@ aliases:
   - "/examples/"
   - "/docs/examples/"
 ---
-
-{{< callout info >}}
```

Comment:
IMO, we can drop this since we never had any feedback and the issue is still in the backlog (we won't lose the information).